### ISSUE TITLE
FastImage isn't avaiable on Expo ? 

### ISSUE THREAD

**Issue by duylinhdang1998** [DylanVann/react-native-fast-image#568]
I using Expo to build RN app and using fastimage lib in my app. But it not working?  
why? Please help me

**Comment by dolsem** [comment]
FastImage is written in native code. Expo's managed workflow does not support adding native code to apps. You need to switch to bare workflow - https://docs.expo.io/versions/latest/introduction/managed-vs-bare

**Comment by tomparkp** [comment]
Expo is working on adding the ability to include native plugins: https://blog.expo.io/expo-managed-workflow-in-2021-d1c9b68aa10

The Expo tooling is miles ahead of whats available for bare RN so it would be awesome to support it once its ready.

**Comment by tomparkp** [comment]
Update - I think it should be possible to support Expo now via a config plugin: https://docs.expo.io/guides/config-plugins/

**Comment by Stevemoretz** [comment]
> Update - I think it should be possible to support Expo now via a config plugin: https://docs.expo.io/guides/config-plugins/

Yes it is, if you check the feature request canny for it they have even explained how.

**Comment by ggepenyan** [comment]
Is there any way to use it with Expo? I really need that. Thanks in advance!

**Comment by alexander01202** [comment]
Hello please add this to expo

**Comment by AliAlican** [comment]
Hello, can we add support for expo managed workflow. using -app.config.js

**Comment by Stevemoretz** [comment]
Expo is supported just use EAS why is this still open?

Brent Vatne from https://expo.canny.io/feature-requests/p/react-native-fast-image:

> Today, we are pleased to announce that with the arrival of [Expo SDK 42](https://blog.expo.io/expo-sdk-42-579aee2348b6), you can now create [custom development clients](https://docs.expo.io/clients/introduction/) in the cloud using [EAS Build](https://docs.expo.io/build/introduction/), or locally with the new [Expo run commands](https://blog.expo.io/introducing-expo-run-commands-835ae8da4813).
You can try it out today:
Create a new project: expo init
Install the packages: expo install react-native-fast-image expo-dev-client
Build the native app:
In the cloud with eas build -p all --profile development (expo build does not support custom builds)
Or locally with expo run:ios and expo run:android (note: see https://expo.fyi/prebuild-cleanup for information on the generated files)
To be clear, react-native-fast-image will not have been added to the Expo Go app that you download from the App Store or Google Play Store.
We still consider this feature to be completed because users can now create a custom client app that includes this library, without needing to write any native code.
We hope you enjoy, and we have big plans to continue improving this workflow going forward!

**Comment by AliAlican** [comment]
Thanks a lot, i guess the issue can be closed, also if we can add the expo install documentation to the readme, that would great

**Comment by AliAlican** [comment]
https://github.com/DylanVann/react-native-fast-image/pull/947

I created this PR
